HHAuto.user.js

```diff
@@ -497,11 +493,18 @@ function gotoPage(page,inArgs,delay = -1)
             togoto = getHHScriptVars("pagesURLShop");
             break;
         case getHHScriptVars("pagesIDQuest"):
+            let mainQuest = getStoredValue("HHAuto_Setting_autoQuest") === "true";
+            let sideQuest = getHHScriptVars("isEnabledSideQuest",false) && getStoredValue("HHAuto_Setting_autoSideQuest") === "true";
             togoto = getHHVars('Hero.infos.questing.current_url');
-            if (togoto.includes("world"))
+            if ((mainQuest && sideQuest && togoto.includes("world")) || (!mainQuest && sideQuest))
+            {
+                togoto = '/side-quests.html';
+            }
+            else if (togoto.includes("world"))
             {
                 logHHAuto("All quests finished, turning off AutoQuest!");
                 setStoredValue("HHAuto_Setting_autoQuest", false);
+                setStoredValue("HHAuto_Setting_autoSideQuest", false);
```

Comment:
if the if is validated it does means sidequest is already off I think

Reply:
right